src/utilities/bcl/test/BCLFixture.cpp

```diff
@@ -64,24 +70,24 @@ void BCLFixture::SetUp() {
 
 void BCLFixture::TearDown() {
 
-  openstudio::filesystem::remove_all(currentLocalBCLPath);
-
+  LocalBCL::close();
+  try {
+    openstudio::filesystem::remove_all(currentLocalBCLPath);
+  } catch (...) {  }
 }
 
-void BCLFixture::SetUpTestCase() {
+void BCLFixture::SetUpTestSuite() {
   // set up logging
   logFile = FileLogSink(toPath("./BCLFixture.log"));
   logFile->setLogLevel(Info);
 }
 
-void BCLFixture::TearDownTestCase() {
+void BCLFixture::TearDownTestSuite() {
   logFile->disable();
 }
 
-std::string BCLFixture::prodAuthKey;
-std::string BCLFixture::devAuthKey;
-
-// these are Dan's API keys labelled under "Testing", delete when there is a better way to do this
-std::string BCLFixture::defaultProdAuthKey("2da842aa2d457703d8fdcb5c53080ace");
-std::string BCLFixture::defaultDevAuthKey("e8051bca77787c0df16cbe13452e7580");
+// define static storage
```

Comment:
prodAuthKey / devAuthKey was moved as a member of the test class instead of being static.